src/routes/+page.svelte

```diff
@@ -60,7 +64,9 @@
 	</section>
 
 	{#each sections as section, index}
-		<section id={`section-${index}`} bind:this={section.el}>
+		<!-- Allow focus jumping to section to avoid focus styles being applied to input on Safari link click -->
+		<!-- svelte-ignore a11y_no_noninteractive_tabindex -->
+		<section id={`section-${index}`} tabindex={-1} bind:this={section.el}>
```

Comment:
In testing I didn't see a reason to have the tabindex here -- is it necessary?

Reply:
Ah yeah I was finding that the "previous" button was only jumping back properly for me with the tabindex set